node/narwhal/src/helpers/committee.rs

```diff
@@ -17,12 +17,37 @@ use snarkvm::console::{prelude::*, types::Address};
 use indexmap::IndexMap;
 use std::collections::HashSet;
 
+#[derive(Clone, Debug, PartialEq)]
+struct CommitteeMembers<N: Network> {
+    /// total stake of all `members`
+    total_stake: u64,
+    /// A map of `address` to `stake`.
+    members: IndexMap<Address<N>, u64>,
+}
+
 #[derive(Clone, Debug, PartialEq)]
 pub struct Committee<N: Network> {
     /// The current round number.
     round: u64,
-    /// A map of `address` to `stake`.
-    members: IndexMap<Address<N>, u64>,
+    /// The current committee members in this `round`
+    committee: CommitteeMembers<N>,
```

Comment:
Maybe better `members`? Otherwise you get `Committee::committee` which looks a bit weird (to me at least :sweat:)

Reply:
I tried that name first, but then the functions like `is_committee_member` code reads like `self.members.members.contains_key(...)` which is weird too :grin:

Reply:
Folded `CommitteeMembers` into `Committee`, so naming is no longer an issue